lib/rubocop/cop/rspec/example_wording.rb

```diff
@@ -53,27 +53,30 @@ def on_block(node)
           end
         end
 
-        def autocorrect(range)
-          ->(corrector) { corrector.replace(range, replacement_text(range)) }
+        def autocorrect(node)
+          lambda do |corrector|
+            corrector.replace(docstring(node), replacement_text(node))
+          end
         end
 
         private
 
         def add_wording_offense(node, message)
-          expr = node.loc.expression
+          add_offense(node, location: docstring(node), message: message)
```

Comment:
It's a bummer that `docstring(node)` is called twice on offending nodes, once when we're adding an offence, and for the second time in auto-correction, but this doesn't seem to be a big deal.

Reply:
If anyone has a code base with enough offenses that this will cause a performance problem, I’m sure they will disable the cop 😉

Reply:
That should be a codebase of an extraordinary size! ;)
```ruby
> t0=Time.now
> 10_000_000.times { Parser::Source::Range.new(expr.source_buffer, expr.begin_pos + 1, expr.end_pos - 1) }
> puts Time.now - t0
5.982299
```
And we're calling it for the second time only when auto-correcting.